cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/ExpandHelper.java

```diff
@@ -25,12 +29,25 @@ public class ExpandHelper {
 
     private final FhirContext fhirContext;
     private final TerminologyServerClient terminologyServerClient;
+    public static final List<String> unsupportedParametersToRemove =
+            Collections.unmodifiableList(new ArrayList<String>(Arrays.asList(Constants.CANONICAL_VERSION)));
```

Comment:
the `canonical-version` expansion parameter is removed until we can get some kind of `capabilitystatement-operation-supported-parameter` extension to fit VSAC's current implementation